### PR TITLE
Filter out blogs without Twitter handle while generating Twitter message

### LINKED ISSUE #457: Incorrect message on Twitter with new issue announcement
Source: https://twitter.com/JVM_bloggers/status/936550555878641665

    Nowy numer #115 już online - http://jvm-bloggers.com/issue/115  z postami m.in. , @softwaremill i @CodeCouple_pl #java #jvm

Problem lies in `m.in <here should be some twitter handle>, @softwaremill (...)`. It looks like there should be a twitter handle there but it wasn't present (maybe random picked an author without twitter handle specified?).

### PR DESCRIPTION
**Pull request by dmydlarz**
Closes #457

### CODE REVIEW

File: src/test/groovy/com/jvm_bloggers/core/social/twitter/TweetContentGeneratorSpec.groovy [hunk 1]
```diff
@@ -178,6 +178,31 @@ class TweetContentGeneratorSpec extends Specification {
...
+            .issueNumber(ISSUE_NUMBER)
+            .heading("issue heading")
+            .blogPosts(blogWithoutTwitter())
+            .build()
+
+        when:
+        String tweetContent = contentGenerator.generateTweetContent(issue)
+
+        then:
+        def handles = /.*m\.in\. @personal\d{1} i @company\d{1}.*/
```
Comment (tdziurko) [inline src/test/groovy/com/jvm_bloggers/core/social/twitter/TweetContentGeneratorSpec.groovy]: `personal1` and `company1` are duplicated here and below in the `blogWithoutTwitter()` method

Comment (dmydlarz) [reply src/test/groovy/com/jvm_bloggers/core/social/twitter/TweetContentGeneratorSpec.groovy]: The same story is for multiple other specs. What other would you expect here?

File: src/test/groovy/com/jvm_bloggers/core/social/twitter/TweetContentGeneratorSpec.groovy [hunk 2]
```diff
@@ -178,6 +178,31 @@ class TweetContentGeneratorSpec extends Specification {
...
+        tweetContent ==~ /$handles/
+    }
+
+    private Collection<BlogPost> blogWithoutTwitter() {
```
[the remark this answers is not in the document]
Comment (dmydlarz) [reply src/test/groovy/com/jvm_bloggers/core/social/twitter/TweetContentGeneratorSpec.groovy]: Changed, but it wasn't easy though. Better?

**Comment by dmydlarz** [comment]
@tdziurko pls let me know if it's acceptable now

**Pull request merged**